app/helpers/bets_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BetsHelper
+  def creator?
+    current_user == @bet.creator
```

Comment:
@bet should a parameter for the reasons mentioned below

1) If @bet is not defined, this would raise an Internal Server Error
2) Assuming that the variable is defined somewhere is an unsafe assumption
3) There is no guarantee that this the data you expect as different parts of the code can set and override each other @bet variable

This should also be a instance method on the Bet model or the User model (for example `@bet.created_by?(user)`)